### PR TITLE
Update about us

### PR DESCRIPTION
**Pull request by EuniceSim142**
Changes made: 

- Add profile pictures in docs/images
- Add skeleton Portfolio pages in docs/team
- Update About us page

closes https://github.com/AY2122S1-CS2103T-W16-2/tp/issues/10

### CODE REVIEW

File: docs/AboutUs.md
Comment (sivayogasubramanian) [inline docs/AboutUs.md:9]: Can we remove the email at line 8?

**Review by sivayogasubramanian — CHANGES_REQUESTED** [review]
LGTM! After removing the email, we can merge.

File: docs/AboutUs.md
Comment (EuniceSim142) [reply docs/AboutUs.md:9]: Sure

**Review by sivayogasubramanian — APPROVED** [review]

**Pull request merged**
